crates/essential-rest-client/src/main.rs

```diff
@@ -114,10 +114,10 @@ async fn run(cli: Cli) -> anyhow::Result<()> {
         } => {
             let builder_client = EssentialBuilderClient::new(builder_address)?;
             let solution = serde_json::from_str::<Solution>(&from_file(solution).await?)?;
```

Comment:
I think this command should be named SubmitSolutionSet and expect a `SolutionSet` instead of a `Solution`, as that was the behaviour before the rename.

```suggestion
            let solution_set = serde_json::from_str::<SolutionSet>(&from_file(solution_set).await?)?;
```